wlct/management/commands/engine.py

```diff
@@ -72,6 +74,103 @@ def parse_and_update_clan_logo():
     except Exception:
         log_exception()
 
+def is_correct_player(player_token, player_team):
+    try:
+        player = Player.objects.filter(token=player_token)[0]
+        tt = TournamentTeam.objects.filter(pk=player_team)[0]
+        if player.clan and player.clan.name == tt.clan_league_clan.clan.name:
+            return True
+
+        tplayers = TournamentPlayer.objects.filter(team=tt)
+        for tplayer in tplayers:
+            if tplayer.player.token == player_token:
+                return True
+        return False
+    except:
+        log_exception()
+        return False
+
+
+def is_valid_team_order(team_order, team_dict, teams):
+    for i in range(len(team_order)):
+        for player in team_dict[team_order[i]]:
+            if is_correct_player(player, teams[i]):
+                return True
+    return False
+
+def get_clan_league_games():
+    cl_games = []
+    clan_leagues = ClanLeague.objects.filter(has_started=True, is_official=True)
+    for cl in clan_leagues:
+        cl_tournaments = ClanLeagueTournament.objects.filter(parent_tournament=cl)
+        for cl_tournament in cl_tournaments:
+            games = TournamentGame.objects.filter(tournament=cl_tournament, is_finished=False)
+            for game in games:
```

Comment:
Nit: No need for the extra variable, can just return the queryset games as you can iterate through those as well the same way.